src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -103,6 +110,91 @@ class IpexCommunicationService extends AgentService {
     });
   }
 
+  async offerAcdc(notificationId: string) {
+    const keriNoti = await this.getNotificationRecordById(notificationId);
+    const msgSaid = keriNoti.a.d as string;
+    const msg = await this.signifyClient.exchanges().get(msgSaid);
+    const schemaSaid = msg.exn.a.s;
+    const creds = await this.signifyClient.credentials().list({
+      filter: {
+        "-s": { $eq: schemaSaid },
```

Comment:
It's not just the schema - apply allows you to also filter by attributes within the ACDC